test/mocha.opts

```diff
@@ -3,4 +3,5 @@
 --check-leaks
 --timeout 10000
 --require ./test/common
+--require ./test/promisePolyfill
```

Comment:
Is it necessary to include the polyfill in the test suite as well? We already use `window.RSVP.Promise` in the tests where we need it.

I guess it would be a great deal nicer to polyfill the native promises with RSVP for the tests, but then I think you should fix up all occurrences of:

```js
    var Promise = typeof weknowhow === 'undefined' ?
        require('rsvp').Promise :
        window.RSVP.Promise;
```

... in the test suite as part of this, and also include the promise polyfill in `test/tests.tpl.html`.